### PR TITLE
added Bliss theme

### PR DESCRIPTION
**Pull request by katacarbix**

### CODE REVIEW

**Comment by AryToNeX** [comment]
you might want to check if it's compatible with powercord, so we make bowoser happy (idk how it handles themes ngl)

**Comment by katacarbix** [comment]
oh you're right, i haven't tested it with anything other than ED yet.

**Comment by AryToNeX** [comment]
still waiting~ have you tested it with other clients?

**Comment by katacarbix** [comment]
nope sorry, been busy.

**Comment by katacarbix** [comment]
`powercord_manifest.json` added so theoretically it should work...

File: discord/bliss/powercord_manifest.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Bliss",
+    "description": "A minimal flat theme with dark and light mode. Meant to be used with Glasscord.",
+    "version": "1.6",
+    "author": "katacarbix",
+    "license": "Apache-2.0",
+    "theme": "theme.scss",
```
Comment (AryToNeX) [inline discord/bliss/powercord_manifest.json:7]: isn't it `css` instead of `scss`???

Comment (katacarbix) [reply discord/bliss/powercord_manifest.json:7]: oh shoot you're right, didn't read it closely enough after i coped it lol

**Comment by katacarbix** [comment]
i should also make sure the meta in the css is correct.

**Comment by AryToNeX** [comment]
please do it~

**Pull request merged**